employee-rostering-backend/pom.xml

```diff
@@ -179,6 +200,7 @@
         <plugin>
           <groupId>org.springframework.boot</groupId>
           <artifactId>spring-boot-maven-plugin</artifactId>
+          <version>2.1.8.RELEASE</version>
```

Comment:
All occurrences of this version string should be replaced with a property (`${version.org.springframework.boot}`).